src/main/java/org/fungover/storm/fileHandler/FileRequestHandler.java

```diff
@@ -0,0 +1,51 @@
+package org.fungover.storm.fileHandler;
+
+import org.fungover.storm.client.HttpParser;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+
+public class FileRequestHandler {
+    private static Path path;
```

Comment:
Since the server is multi threaded it's not a good idea to use static variables to store this type of information. We can use static methods so we don't have to create a new FileRequestHandler each time we handle a connection but then we can only rely on local variables and objects created inside the methods.
I suggest returning an Object holding file and filesize from handleRequest method.

I think someone is working on some objekt for representing a httpresponse which could be used as soon as that is merged. Issue #35 and Pullrequest #58